bin/install

```diff
@@ -1,85 +1,59 @@
 #!/usr/bin/env bash
 
-set -o nounset -o pipefail -o errexit
+set -eu -o pipefail
 
 # shellcheck source=../lib/utils.sh
-source "$(dirname "$0")/../lib/utils.sh"
-
-NODEJS_CHECK_SIGNATURES="${NODEJS_CHECK_SIGNATURES:-strict}"
-
+source "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/../lib/utils.sh"
```

Comment:
Why did this line need to change? We are using `source "$(dirname "$0")/utils.sh"` in the Erlang plugin.

Reply:
I'm not 100% sure actually, I think I'm using `${BASH_SOURCE[0]}` because I had some problems with `$0` and symlinks. The `$(cd ... && pwd)` is for normalizing relative paths. But I don't they really matters when being called by `asdf`, I will revert this change